components/Footer.js

```diff
@@ -1,23 +1,44 @@
 import Link from "next/link";
 import styled from "styled-components";
+import { Icon } from "@iconify/react";
 
 export default function Footer() {
-  return <StyledLink href="/create">+ Add new</StyledLink>;
+  return (
+    <StyledFooter>
+      <StyledLink href="/create">+ Add new</StyledLink>
+      <StyledLink href="/favorite">
+        {" "}
+        <Icon
+          icon="ic:sharp-star"
+          width="24"
+          strokeWidth="1.5"
+          stroke="var(--white"
+        />
+        {"  "}
+        Favorites
+      </StyledLink>
+    </StyledFooter>
+  );
 }
 
-const StyledLink = styled(Link)`
+const StyledFooter = styled.div`
```

Comment:
We find it more intuitive to call the component Footer since it is primarily a footer even it has a navigation functionality. But we changed the div to nav.